### DISCUSSION TITLE
How to launch a command as non blocking call?

### DISCUSSION THREAD

**Discussion by ritulahkar** [nushell/nushell#11882] [Q&A]
In gnome terminal `xed /var/log/rkhunter.log &` will non blocking but I can not execute `config env &` in nushell. How to achieve the same?

**Comment by fdncred** [comment]
Job management is not supported at this time in nushell. It's on the list of todos.

**Comment by ghost** [comment]
as fdncred said its not yet in nushell, but external programs can add it.
the cookbook used to recommend pueue for it.
i use a bit of extra config to make it more usable: <https://github.com/Jan9103/nushell_pueue/blob/main/job.nu> put this file somewhere and load it in your `config.nu` via `use /path/to/job.nu` (the filename is the resulting command name).
the additional config makes it possible to pass nu codeblocks (with syntax highlighting, tab completion, etc) instead of just bash command strings to it. it also converts returned values into nu objects

an alternative could be [nq](https://github.com/leahneukirchen/nq) (very minimal) or some sort of alias to systemd-run/abduco/… with args to detach it

**Comment by Mrfiregem** [comment]
You can now use `job spawn { xed /var/log/rkhunter.log }` to run commands in the background.
